### DISCUSSION TITLE
[灌水]好想和匿叶龙那啥啊……

### DISCUSSION THREAD

**Discussion by NEBrightULA** [share121/inter-knot#4519] [原神]
匿叶龙🥵🥵🥵
![1725182597233](https://github.com/user-attachments/assets/f3868244-32b4-48b4-8665-d368c1aa0881)
![1725182544319](https://github.com/user-attachments/assets/fffc008d-d3b3-4484-ba74-474a3706f3b5)

**Comment by March-777th** [comment]
你也逆天<img src="https://bbs-static.miyoushe.com/static/2024/07/31/73346f2b98d45b62da9139d2774fa856_8641114165088315754.png" alt="三月七 告退" width="75" >

**Reply by laodeng2** [reply]
无敌了

**Comment by happyfucking** [comment]
Σ(°Д°;逆天

**Comment by arlcius** [comment]
逆天！

**Comment by SeeleVollereid** [comment]
逆天
![img717378057_1722497871357](https://github.com/user-attachments/assets/eb1e0d18-dec6-43a0-81ee-7755b64ef9c4)

**Comment by Xyy12345689** [comment]
逆天Σ(ŎдŎ|||)ﾉﾉ

**Comment by 001kd** [comment]
人不能……至少不应该……

**Reply by NEBrightULA** [reply]
人可以，至少得试试…

**Comment by summer293** [comment]
（｀Δ´）！逆天

**Comment by QvQ-ED** [comment]
好好好

**Comment by ysdtal** [comment]
逆天

**Comment by mis-sea** [comment]
逆天

**Comment by tuhfe3-piWvex-cupjin** [comment]
大哥，这可不兴说啊

**Comment by Yotsuba-Ciel** [comment]
哥，你想想就好了，不要付诸行动

**Reply by NEBrightULA** [reply]
现实中没有匿叶龙，心有余而力不足啊…

**Comment by dzlf-10101** [comment]
尾巴怎么玩

**Reply by Dingdong1018** [reply]
就...那么玩（）

**Comment by ncepugaq** [comment]
好得很

**Comment by Power-Bai** [comment]
逆天

**Comment by wgvshyshwbhshhbeb** [comment]
逆天

**Comment by wum2008** [comment]
逆天

**Reply by SAJN8455** [reply]
6

**Comment by nzj2004** [comment]
逆天了哥们

**Comment by ssbohet** [comment]
不是哥们，你是真饿啊

**Comment by ID104772508** [comment]
逆天

**Comment by Noxiil092333** [comment]
逆天

**Comment by lllliufeng** [comment]
知天易

**Reply by Yizakl** [reply]
逆天男

**Comment by ID104772508** [comment]
逆天<img src="https://img-static.mihoyo.com/communityweb/upload/2bc65c226456bea0398f59387142d6cf.png" alt="帕姆 震惊" width="75" >

**Comment by mr-xiyu** [comment]
**逆大天**

**Comment by ijog36** [comment]
逆天

**Comment by fhw010129** [comment]
人不能，也不应该……
逆天

**Comment by Ritomm** [comment]
逆天

**Comment by 596wjr** [comment]
逆天

**Comment by love262-wkx** [comment]
人不能操龙，至少不应该操

**Comment by xiangzhexch** [comment]
逆天

**Comment by bushigemengsishuihaiyinga** [comment]
▼△▼逆天（尾巴音）

**Comment by beiya0221** [comment]
原来不止我觉得匿叶龙有一种奇怪的涩气，就是很涩啊很涩

**Reply by bushigemengsishuihaiyinga** [reply]
你也逆天

**Reply by NEBrightULA** [reply]
确实啊，果然是有人能看到匿叶龙的身姿曼妙之处的！

**Reply by fzdqz** [reply]
你也发配印度

**Comment by 129776** [comment]
逆天
![comment_49606aae8e7e43bcf7fac8294f42fda5](https://github.com/user-attachments/assets/4aaa575a-9e66-4660-b802-bd287b5c2e0c)

**Comment by Letyem** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_4941628593528507734.png" alt="阿姬倒地了03" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_4941628593528507734.png" alt="阿姬倒地了03" width="75" >逆天

**Comment by yuzhoujimengxin** [comment]
顶级

**Comment by Junjun280** [comment]
逆大天

**Comment by Usodaru48763** [comment]
我咧個人壽

**Comment by xinghenlongJR** [comment]
逆天

**Comment by Lun603** [comment]
兄弟...別...別這樣<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >

**Comment by keyuanya** [comment]
我是卖保险的，这就是人寿

**Comment by yuunnnnnnn** [comment]
我也想🥵

**Comment by For-elysia** [comment]
你小子
![1727025041132](https://github.com/user-attachments/assets/11523c30-7555-4c88-9e7b-4c7fa96c07b2)

**Comment by fzdqz** [comment]
发配印度。

**Comment by nabeilishi** [comment]
呃呃...逆天

**Comment by 210flight** [comment]
知天易

**Comment by zlzdg** [comment]
逆天

**Comment by hoyotao** [comment]
孽龙

**Comment by ususwcsm** [comment]
逆天

**Comment by Oumainory** [comment]
我也想<img src="https://bbs-static.miyoushe.com/static/2024/09/04/a9fbeee1e21cf1abcf7bf2e610ee91f5_8437422338490524317.png" alt="玛拉妮-气氛组" width="75" >

**Comment by Mrbeichen** [comment]
**逆天。。。**

**Comment by XiaoYumuamua** [comment]
逆天

**Comment by WL-BLACK** [comment]
我嘞个![PSX_20241004_161949.jpg](https://github.com/user-attachments/assets/1f9e3635-7c8a-4a68-a307-88f29ad52790)

**Comment by 11lookcat** [comment]
cc你的

**Comment by 129776** [comment]
> 匿叶龙🥵🥵🥵 ![1725182597233](https://private-user-images.githubusercontent.com/180718809/365333105-f3868244-32b4-48b4-8665-d368c1aa0881.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI5MDc3MTgsIm5iZiI6MTczMjkwNzQxOCwicGF0aCI6Ii8xODA3MTg4MDkvMzY1MzMzMTA1LWYzODY4MjQ0LTMyYjQtNDhiNC04NjY1LWQzNjhjMWFhMDg4MS5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyOVQxOTEwMThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZjM4NTEzZThjNGQwZjI0ZjM0YjNhNTA4M2ZkMDZlMGNmZjlmMDBhN2I1NGM4NWNmNWRhM2EyYzMyZThlYzkzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ceMrm3sO-qO7N3XSLsDZiPfqoEL9Htbe9BaDOkzsUQs) [ ![1725182597233](https://private-user-images.githubusercontent.com/180718809/365333105-f3868244-32b4-48b4-8665-d368c1aa0881.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI5MDc3MTgsIm5iZiI6MTczMjkwNzQxOCwicGF0aCI6Ii8xODA3MTg4MDkvMzY1MzMzMTA1LWYzODY4MjQ0LTMyYjQtNDhiNC04NjY1LWQzNjhjMWFhMDg4MS5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyOVQxOTEwMThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZjM4NTEzZThjNGQwZjI0ZjM0YjNhNTA4M2ZkMDZlMGNmZjlmMDBhN2I1NGM4NWNmNWRhM2EyYzMyZThlYzkzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ceMrm3sO-qO7N3XSLsDZiPfqoEL9Htbe9BaDOkzsUQs) ](https://private-user-images.githubusercontent.com/180718809/365333105-f3868244-32b4-48b4-8665-d368c1aa0881.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI5MDc3MTgsIm5iZiI6MTczMjkwNzQxOCwicGF0aCI6Ii8xODA3MTg4MDkvMzY1MzMzMTA1LWYzODY4MjQ0LTMyYjQtNDhiNC04NjY1LWQzNjhjMWFhMDg4MS5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyOVQxOTEwMThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZjM4NTEzZThjNGQwZjI0ZjM0YjNhNTA4M2ZkMDZlMGNmZjlmMDBhN2I1NGM4NWNmNWRhM2EyYzMyZThlYzkzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ceMrm3sO-qO7N3XSLsDZiPfqoEL9Htbe9BaDOkzsUQs) [ ](https://private-user-images.githubusercontent.com/180718809/365333105-f3868244-32b4-48b4-8665-d368c1aa0881.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI5MDc3MTgsIm5iZiI6MTczMjkwNzQxOCwicGF0aCI6Ii8xODA3MTg4MDkvMzY1MzMzMTA1LWYzODY4MjQ0LTMyYjQtNDhiNC04NjY1LWQzNjhjMWFhMDg4MS5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyOVQxOTEwMThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZjM4NTEzZThjNGQwZjI0ZjM0YjNhNTA4M2ZkMDZlMGNmZjlmMDBhN2I1NGM4NWNmNWRhM2EyYzMyZThlYzkzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ceMrm3sO-qO7N3XSLsDZiPfqoEL9Htbe9BaDOkzsUQs) ![1725182544319](https://private-user-images.githubusercontent.com/180718809/365333125-fffc008d-d3b3-4484-ba74-474a3706f3b5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI5MDc3MTgsIm5iZiI6MTczMjkwNzQxOCwicGF0aCI6Ii8xODA3MTg4MDkvMzY1MzMzMTI1LWZmZmMwMDhkLWQzYjMtNDQ4NC1iYTc0LTQ3NGEzNzA2ZjNiNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyOVQxOTEwMThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hOWI4MDZiNWJlNTA5NDcxZDZlYWViMmJhOTQ2NDlmZTZhMzcwMTkyMDJkZjdlYjAyZGI5YzgzZmE5ZGQ2NmM1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.1FrRvd56WuiVnTOSdUM-bgpaH9rTDEVD-1o0-U5G5ss)

。

**Comment by 129776** [comment]
炸裂

**Comment by Akanyi** [comment]
逆天

**Comment by he222111** [comment]
逆天

**Comment by Mark-37** [comment]
人不能，至少不应该

**Comment by Kevin-Kaslanna** [comment]
逆天🤙

**Comment by Admirevegaa** [comment]
人之常情

**Comment by duzuochuangaokanyanyu** [comment]
逆天

**Comment by xiayu-u** [comment]
逆大天

**Comment by Rjjfk** [comment]
逆天啊
![Uploading Image_118116688913746.jpg…]()

**Comment by internet3304** [comment]
逆天

**Comment by bxzwq** [comment]
逆天

**Comment by friedericus1848** [comment]
纳塔有一套完整的法律保护各龙众不受侵害，朋友我劝你好自为之

**Comment by Kiana175** [comment]
你有点太过逆天了，跟本小姐做那啥，都比跟匿叶龙那啥好！

**Comment by yss161** [comment]
饿急了

**Comment by LouisVento** [comment]
虾头

**Comment by Acheron777** [comment]
人之常情

**Reply by yss161** [reply]
互相理解

**Comment by sg15012291951** [comment]
逆天

**Comment by c328422** [comment]
谷歌搜尋：一女子讓狗舔她的...  簡直就是你敢想，就已經一個人實踐過，不過是狗跟女性的炸裂新聞

**Comment by Loxia1121** [comment]
烟迷主那里的龙宝宝也很可爱（会追着自己的尾巴转圈圈，还会好奇的看着你（一双耳朵还duangduang的 轻轻的上下懂 像rua（）

**Comment by zsj200** [comment]
逆天

**Comment by 323157364** [comment]
逆天

**Comment by lcp1234** [comment]
通天

**Comment by fzdqz** [comment]
是个人物

**Comment by baimo2004** [comment]
逆天

**Comment by Lz-zzz** [comment]
为什么偏偏是匿叶龙？（邦布-好奇）
呜呜呜！绫人歪一斗了！怒🌿大伟丘！
![mmexport1748195058501](https://github.com/user-attachments/assets/62f4838e-4805-4d95-9891-511285991720)

**Comment by LeiK95** [comment]
……或许可以……

**Reply by Dingdong1018** [reply]
不可以也得可以（

**Comment by WWWIIINNN114** [comment]
兄弟你是真的饿了

**Comment by kiana114** [comment]
逆天………

**Comment by l-wenyu-I** [comment]
![IMG_20250430_060141](https://github.com/user-attachments/assets/f7bc5358-2515-4b2e-85b9-42901e07e930)
布什，戈门

**Comment by qinYun256** [comment]
不是哥们,jpg

**Comment by YuanTu-tanosii** [comment]
邦——（逆——）布——（天——）

**Comment by AS111552** [comment]
戈门你

**Comment by wanggary09487** [comment]
你牛逼

**Comment by jianci57** [comment]
逆天...
